app/utils/APIUtils.js

```diff
@@ -94,17 +95,18 @@ function getSuccessPayload(options) {
 }
 
 function getSuccessTypeDescriptor(type, options = {}) {
+  const meta = options.meta || {};
```

Comment:
- [ ] This is actually not needed. `Object.assign` will work even if options.meta is `undefined` or `null`.